controllers/k8ssandra/datacenters.go

```diff
@@ -475,3 +459,13 @@ func dcUpgradePriority(dc api.CassandraDatacenterTemplate) int {
 
 	return 2
 }
+
+func (r *K8ssandraClusterReconciler) removeRebuildDcAnnotation(kc *api.K8ssandraCluster, ctx context.Context) result.ReconcileResult {
```

Comment:
This refactoring is probably unnecessary, since it was done when I thought I'd need to remove the annotation for another feature. It was not the case in the end.
It's a valuable refactoring though to reduce the cognitive complexity of the `reconcileDcRebuild()` function.